### PR TITLE
Add copyright to bootstrap.properties

### LINKED ISSUE #14429: Add Copyright to bootstrap.properties in JSF 2.3 FAT
Servlet 4.0 FAT may also need them too

### PR DESCRIPTION
**Pull request by volosied**
fixes  #14429

Tried to keep dates as accurate as I could.

Majority of the files were from 2017.

### CODE REVIEW

**Comment by volosied** [comment]
#build

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/bootstrap.properties [hunk 1]
```diff
@@ -1,3 +1,13 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/bootstrap.properties:3]: Should this one be 2020?

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_PushBuilderSecurity/bootstrap.properties [hunk 2]
```diff
@@ -1,3 +1,13 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_PushBuilderSecurity/bootstrap.properties:3]: 2018? https://github.com/OpenLiberty/open-liberty/pull/3112

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/bootstrap.properties (hunk 1 above)
Comment (volosied) [reply dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_SCIException/bootstrap.properties:3]: Yep -- #13758
Will correct.

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_PushBuilderSecurity/bootstrap.properties (hunk 2 above)
Comment (volosied) [reply dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_PushBuilderSecurity/bootstrap.properties:3]: Will correct as well.

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#run-libby-bot

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#run-libby-bot

**Review by pnicolucci — APPROVED** [review]

**Pull request merged**